### PR TITLE
localization updates

### PR DESCRIPTION
**Pull request by shermandickman**

### CODE REVIEW

**Comment by shermandickman** [comment]
@mkaply Can you take a look at message size in messenger.properties?  It was displaying incorrectly in Spanish, and the test of these files seemed to adopt this approach.

**Comment by shermandickman** [comment]
@mkaply Can you review please?

**Comment by mkaply** [comment]
Sorry, didn't get notified about this. Looking.

File: es-ES/mail/chrome/messenger/messenger.properties [hunk 1]
```diff
@@ -424,10 +424,12 @@ editVirtualFolderPropertiesTitle=Editar las propiedades de la búsqueda %S
 #alert to inform the user to choose one or more folders to search for a saved search folder
 alertNoSearchFoldersSelected=Debe elegir al menos una carpeta en la que buscar para guardar la búsqueda.
 
-# These two are displayed in the message and folder pane windows
-# LOCALIZATION NOTE(????ByteAbbreviation): Do not translate %d below, it is the size of the message/folder
-kiloByteAbbreviation=%dKB
-megaByteAbbreviation=%dMB
+# These are displayed in the message and folder pane windows
+# LOCALIZATION NOTE %.*f is the abbreviated size in the appropriate units
+byteAbbreviation2=%.*f bytes
```
Comment (mkaply) [inline es-ES/mail/chrome/messenger/messenger.properties]: You don't want to replace these, you just want to add the Abbreviation2 versions. Removing the kilo/mega will break other places.

We're also missing giga

https://github.com/postbox/postbox/blob/4f3935c82a440eb6b622bac2015027f309afc192/mail/locales/en-US/chrome/messenger/messenger.properties#L443

We should match  English

Comment (shermandickman) [inline es-ES/mail/chrome/messenger/messenger.properties]: Note that all messenger.properties files seem to do something different. There was an issue with es-ES using the old method.  The attached is what master looks like:

screen shot 2017-03-23 at 8 54 51 am

Comment (mkaply) [inline es-ES/mail/chrome/messenger/messenger.properties]: I don't know why %d wouldn't work. That's standard printf stuff. As long as it matches the others, it should work. Problem here is we didn't want to remove the 2 versions.

Did you test this new version and it worked?

Comment (shermandickman) [inline es-ES/mail/chrome/messenger/messenger.properties]: I did test the new version, and it worked, but I don't know if it broke anything.  

Can you try with your fix?

**Pull request merged**